src/app/models/resource-details.ts

```diff
@@ -1,27 +1,27 @@
-export class ResourceDetails {
-  mal_id!: number;
-  url!: string;
-  images!: any;
-  title!: string;
-  title_english!: string;
-  title_japanese!: string;
-  title_synonyms!: string[];
-  type!: string;
-  status!: string;
-  score!: number;
-  scored_by!: number;
-  rank!: number;
-  popularity!: number;
-  members!: number;
-  favorites!: number;
-  synopsis!: string;
-  background!: string;
-  genres!: Genre[];
+export interface ResourceDetails {
```

Comment:
En otros componentes se usa el término _Item_ en lugar de _Resource_, por consistencia sugiero:

```suggestion
export interface ItemDetails {
```